docs/troubleshoot/cli/use-individual-troubleshoot-commands.md

```diff
@@ -91,28 +88,28 @@ npm --version
 ```
 
 ### npm configuration
-If you are having trouble installing Zowe CLI from an npm registry, gather your npm configuration to help identify issues with registry settings, global install paths, proxy settings, etc...
+If you are having trouble installing Zowe CLI from an npm registry, gather your npm configuration to help identify issues with registry settings, global install paths, proxy settings, etc.:
 
 ```
 npm config ls -l
 ```
 
 ### npm log files
-In case of errors, npm creates log files in the `npm_cache\_logs` location. To get the `npm_cache` location for a specific OS, run the following command:
+In case of errors, npm creates log files in the `npm_cache\_logs` location. To get the `npm_cache` location for a specific OS:
 
 ```
 npm config get cache
 ```
 
-By default, npm keeps only 10 log files, but sometimes more are needed. Increase the log count by issuing the following command:
+By default, npm keeps only 10 log files, but sometimes more are needed. To increase the log count:
 
 ```
 npm config set logs-max 50
 ```
 
-This command increases the log count to 50, so that more log files will be stored on the system. Now you can run tests multiple times and not lose the log files. The logs can be passed to Support for analysis.
+        - This command increases the log count to 50, so that more log files will be stored on the system. Now you can run tests multiple times and not lose the log files. The logs can be passed to Support for analysis.
 
-As the log files are created only when an npm command fails, but you are interested to see what is executed, you can increase the log level of npm. Issue the following command:
+Log files are created only when an npm command fails. To see what is executed, increase the log level of npm:
```

Comment:
Would change to something like: "By default, log files are created only when..."